docs/testing/regression-checklist/mapping.json

```
@@ -28,5 +28,9 @@
 	{
 		"regex": "extensions/blocks/",
 		"testFile": [ "extensions/blocks/checklist.md" ]
+	},
+	{
+		"regex": "^(extensions/blocks/(subscriptions|premium-content)/|extensions/shared/memberships.js)",
```

I think you may have to update that regex since the full path is `projects/plugins/jetpack/extensions...`

Maybe something like this would work better?

```suggestion
		"regex": "^projects\/plugins\/jetpack\/extensions\/(blocks\/(subscriptions|premium-content)\/|shared\/memberships\.js)",
```